visualization/settings/config.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass, field
+from typing import Dict, Optional, Tuple
+
+import hydra
+
+from nemo_skills.code_execution.utils import CODE_OUTPUT_SEPARATORS, CODE_SEPARATORS
+from nemo_skills.inference.generate_solutions import GenerateSolutionsConfig
+
+
+@dataclass
+class BaseVisualizationConfig:
+    prediction_jsonl_files: Dict[str, str] = field(default_factory=dict)
```

Comment:
why is this a dict? Please use different name if it's not the same as prediction_jsonl_files in other configs